crates/interpreter/src/module.rs

```diff
@@ -54,7 +55,8 @@ impl<'m> Module<'m> {
     pub fn new(binary: &'m [u8]) -> Result<Self, Error> {
         let side_tables = validate(binary)?;
         let mut module = unsafe { Self::new_unchecked(binary) };
-        module.side_tables = side_tables;
+        // TODO(dev/fast-interp): We should take a buffer as argument to write to.
```

Comment:
Could you elaborate on why the TODO here is better than the Box solution below? Thanks.

Reply:
The Box solution is just a workaround for testing because it's leaking memory. Ultimately we want a solution that we can use in production. It's not yet clear how it will look like. It could be `Module::new()` taking a buffer to write to and `new_unchecked()` a buffer to read from, or it could be something else. It depends on the "streaming" design. But the buffer solution could be a temporary solution until the streaming design is clear.